providers/src/airflow/providers/google/cloud/operators/vertex_ai/custom_job.py

```diff
@@ -1750,7 +1750,6 @@ class ListCustomTrainingJobOperator(GoogleCloudBaseOperator):
         "region",
         "project_id",
         "impersonation_chain",
-        "display_name",
```

Comment:
It was recently added in https://github.com/apache/airflow/pull/43028/files
could it be that the intent was to add this field to the init but was missed?
cc @WSHoekstra as the original author

Reply:
Adding a field only to the `template_fields` breaks the operator. And we believe that this new field "display_name" isn't needed here. @MaksYermak , please correct me if I'm wrong.

Reply:
@eladkal @moiseenkov you're right, this needs to go.